src/asserts/possible-phone-assert.js

```diff
@@ -0,0 +1,67 @@
+'use strict';
+
+/**
+ * Module dependencies.
+ */
+
+import { PhoneNumberUtil } from 'google-libphonenumber';
+import { Violation } from 'validator.js';
+import { find } from 'lodash';
+
+/**
+ * Phone util instance.
+ */
+
+const phoneUtil = PhoneNumberUtil.getInstance();
+
+/**
+ * Export `PossiblePhoneAssert`.
+ */
+
+export default function possiblePhoneAssert(code) {
+  /**
+   * Optional peer dependencies.
+   */
+
+  const countries = require('isoc');
+
+  /**
+   * Class name.
+   */
+
+  this.__class__ = 'PossiblePhone';
+
+  // Test by `alpha-3` code.
+  let country = find(countries, { alpha3: code });
+
+  // Test by `alpha-2` code.
+  if (!country) {
+    country = find(countries, { alpha2: code });
+  }
+
+  if (!country) {
+    throw new Error('A valid ISO-3166 country code must be specified');
```

Comment:
If we only accept _alpha2_, we might not even need to perform this check. We could simply pass it to `phoneUtil.getRegionCodeForNumber()` if it already does error handling.